pom.xml

```diff
@@ -39,52 +50,26 @@
         <rwandaprimarycareVersion>2.1.0-SNAPSHOT</rwandaprimarycareVersion>
         <rwandareportsVersion>3.0.0-SNAPSHOT</rwandareportsVersion>
-        <serializationVersion>0.2.15</serializationVersion>
+        <serializationVersion>0.2.16</serializationVersion>
         <simplelabentryVersion>2.0.0</simplelabentryVersion>
+        <spaVersion>1.0.9</spaVersion>
         <syncVersion>3.0.0-SNAPSHOT</syncVersion>
+        <uicommonsVersion>2.23.0</uicommonsVersion>
+        <uiframeworkVersion>3.23.0</uiframeworkVersion>
+        <uilibraryVersion>2.0.7</uilibraryVersion>
         <usagestatisticsVersion>2.1.0</usagestatisticsVersion>
         <validationVersion>2.0.0</validationVersion>
-        <webservicesRestVersion>2.36.0</webservicesRestVersion>
-        <spamodulesVersion>1.0.9-SNAPSHOT</spamodulesVersion>
-        <fhir2Version>1.3.0</fhir2Version>
-        <pathologyappVersion>1.1.1-pre.31</pathologyappVersion>
-        <legacyuinavbarappVersion>1.0.1-pre.1</legacyuinavbarappVersion>
-        <imbpathologyrequestVersion>1.0.0-SNAPSHOT</imbpathologyrequestVersion>
-        <patientchartappVersion>3.2.1-pre.85</patientchartappVersion>
-        <fromentryappVersion>3.2.1-pre.85</fromentryappVersion>
-        <genericpatientwidgetsappVersion>3.2.1-pre.85</genericpatientwidgetsappVersion>
-        <patientallergiesappVersion>3.2.1-pre.85</patientallergiesappVersion>
-        <patientappointmentsappVersion>3.2.1-pre.85</patientappointmentsappVersion>
-        <patientattachmentsappVersion>3.2.1-pre.85</patientattachmentsappVersion>
-        <patientbannerappVersion>3.2.1-pre.85</patientbannerappVersion>
-        <patientbiomentricsappVersion>3.2.1-pre.85</patientbiomentricsappVersion>
-        <patientcommonlibVersion>3.2.1-pre.97</patientcommonlibVersion>
-        <patientconditionsappVersion>3.2.1-pre.85</patientconditionsappVersion>
-        <patientformsappVersion>3.2.1-pre.85</patientformsappVersion>
-        <patientimmunizationsappVersion>3.2.1-pre.85</patientimmunizationsappVersion>
-        <patientmedicationsappVersion>3.2.1-pre.85</patientmedicationsappVersion>
-        <patientnotesappVersion>3.2.1-pre.85</patientnotesappVersion>
-        <patientprogramsappVersion>3.2.1-pre.85</patientprogramsappVersion>
-        <patienttestresultsappVersion>3.2.1-pre.85</patienttestresultsappVersion>
-        <patientvitalsappVersion>3.2.1-pre.85</patientvitalsappVersion>
-        <primarynavigationappVersion>3.4.1-pre.82</primarynavigationappVersion>
-        <implementertoolsappVersion>3.4.1-pre.85</implementertoolsappVersion>
-        <devtoolsappVersion>3.3.1</devtoolsappVersion>
-        <loginappVersion>3.4.1-pre.102</loginappVersion>
-        <homeappVersion>3.1.1-pre.81</homeappVersion>
-        <patientsearchappVersion>3.1.1-pre.184</patientsearchappVersion>
-        <oncologypatientstatusVersion>1.1.0</oncologypatientstatusVersion>
-        <fastdataentryappVersion>1.0.0-pre.21</fastdataentryappVersion>
```

Comment:
For the most part, it didn't seem like any of these were actually used.  If we want to include them, just put them directly into the spa-build-config.json file to be more explicit.